CHANGES.rst

```diff
@@ -5,6 +5,13 @@ Changes For IRIS Service
 unreleased
 ==========
 
+ - Extend filters for /admin/petitions by city.portal.id and city.tags
+ - Extend filters for /admin/cities by portal.id and city.tags
+   DEPLOY:
+        - ALTER TABLE cities ADD COLUMN portal['id'] STRING
+        - trigger iris city sync via az.datasync
+            Sync: curl localhost:11600/api/v1/update/iris.cities
```

Comment:
datasync is correctly triggered in ls.scripts: `bin/fab azr.trigger_datasync:<env>,iris.cities`